testing/matrix/linter/rules/modules/modules.go

```diff
@@ -167,8 +167,18 @@ func GetDeckhouseModulesWithValuesMatrixTests() ([]utils.Module, error) {
 		modulesPaths = append(modulesPaths, result...)
 	}
 
+	hasFocusNames := len(focusNames) > 0
 	var lintRuleErrorsList errors.LintRuleErrorsList
 	for _, modulePath := range modulesPaths {
+		if hasFocusNames {
```

Comment:
This check is not required. Even if the map is empty, it works fine, so there will be fewer clauses in the code without this line.